src/hugo/install.sh

```diff
@@ -106,7 +106,14 @@ if ! hugo version &> /dev/null ; then
         arch="64bit"
     fi
 
-    hugo_filename="hugo_${VERSION}_Linux-${arch}.tar.gz"
+    # Install extended version of hugo if desired
+    if [ "${EXTENDED}" = "true" ]; then
+        extended="extended_"
+    else
+        extended=""
+    fi
+
+    hugo_filename="hugo_${extended}${VERSION}_Linux-${arch}.tar.gz"
```

Comment:
Could we add a test (`scenarios.json`) for this? You could use https://github.com/devcontainers/features/tree/main/test/node for reference.

Let me know if you have any questions, happy to walk you through it.

Reply:
Ideally, in https://github.com/devcontainers/features/tree/main/test/hugo

You'd need to create `scenarios.json` with 

```
{
    "install_extended_hugo": {
        "image": "mcr.microsoft.com/devcontainers/base",
        "features": {
            "hugo": {
                "version": "latest",
                "extended": true
            }
        }
    }
}
```

Then create a `install_extended_hugo.sh` file with a test which ensure that hugo's extended version is installed.